### ISSUE TITLE
Using knex and strong-oracle insert a record, but not commit immediately, but only using strong-oracle ok 

### ISSUE THREAD

**Issue by diversoft** [tgriesser/knex#814]
Using knex (using strong-oracle driver) insert a record, but not commit, how to auto commit after insert? 
```javascript
var knexconfig={
  client: 'strong-oracle',
  connection:{
    host     : 'localhost',
    user     : 'testoracle',
    password : 'testoracle',
    database : 'XE'
  },
  migrations: {
    tableName: 'migrations'
  },
  pool: {
    min: 0,
    max: 7
  }
};
var knex = require('knex')(knexconfig);
knex.insert({PROPERTYNAME79: '79',PROPERTYNAME90:'90',PROPERTYNAME38:'38'}, 'PROPERTYNAME79').into('NEWMODEL')
        .catch(function(error) {
            console.error(error);
        }).then(function() {
           return knex.select('*').from('NEWMODEL');
        }).then(function(rows) {
            console.log(rows);
        }).catch(function(error) {
            console.error(error);
        });
```

**Comment by diversoft** [comment]
using transaction ok

```javascript
    knex.transaction(function(trx) {
        knex.table('NEWMODEL').insert({PROPERTYNAME79: '79', PROPERTYNAME90: '90', PROPERTYNAME38: '38'})
            .transacting(trx)
            .then(trx.commit)
            .then(function () {
                return knex.select('*').from('NEWMODEL');
            }).then(function (rows) {
                res.json(rows);
            }).catch(function (error) {
                console.error(error);
            });
    });

**Comment by vschoettke** [comment]
What version of strong-oracle are you using?

**Comment by diversoft** [comment]
"version": "1.6.0"

**Comment by vschoettke** [comment]
This is a strong-oracle problem and was fixed in 1.6.1. See strongloop/strong-oracle#27

**Comment by diversoft** [comment]
ok, thank you!
